libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineView.kt

```diff
@@ -142,14 +145,32 @@ class MapboxRouteLineView(options: MapboxRouteLineOptions) {
      * the layers if they have not yet been initialized. If you have a use case for initializing
      * the layers in advance of any API calls this method may be used.
      *
+     * If the layers already exist they will be removed and re-initialized with the options provided.
+     *
      * Each [Layer] added to the map by this class is a persistent layer - it will survive style changes.
      * This means that if the data has not changed, it does not have to be manually redrawn after a style change.
      * See [Style.addPersistentStyleLayer].
      *
      * @param style a valid [Style] instance
      */
     fun initializeLayers(style: Style) {
-        MapboxRouteLineUtils.initializeLayers(style, options)
+        resetLayers(style)
```

Comment:
```suggestion
        rebuildSourcesAndLayersIfNeeded(style)
```
Let's revert to previous behavior, we don't need to clear the layers to ensure they are initialized.

Reply:
What do you mean by this? The layers and sources must be removed in order to add the layers and sources.

Reply:
Layers and source do not need to be removed every single time `initializeLayers(style: Style)` is called. If they've been already initialized with the correct options, why would we do that?